web/viewer.js

```diff
@@ -901,6 +902,12 @@ var PDFViewerApplication = {
                      store.get('zoom', self.pdfViewer.currentScale);
           var left = store.get('scrollLeft', '0');
           var top = store.get('scrollTop', '0');
+          rotation = store.get('rotation', '0');
+
+          self.setScale(DEFAULT_SCALE, true);
```

Comment:
> Can this be considered a viable solution?

In my opinion, no. This patch tries to add a minor feature, hence I don't think that unconditionally triggering an unnecessary rendering operation during *every* document load is a particularly good thing to do.

Given that the error isn't really a breaking error, but just a console message, I'd *remove* this line and try something else instead: Try changing https://github.com/mozilla/pdf.js/blob/master/web/pdf_viewer.js#L201 to
```javascript
if (this._currentScale !== UNKNOWN_SCALE)
  this._setScale(this._currentScaleValue, true);
}
```

*If this seem to work: please make sure that you test the patch __thoroughly__, to ensure first of all that it does not regress other functionality, and second of all that the rotation works as intended!*